docs/apache-airflow-providers-microsoft-azure/operators/sftp_to_wasb.rst

```diff
@@ -0,0 +1,67 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Azure Blob Storage Transfer Operator
+====================================
+The Blob service stores text and binary data as objects in the cloud.
+The Blob service offers the following three resources: the storage account, containers, and blobs.
+Within your storage account, containers provide a way to organize sets of blobs.
+For more information about the service visit `Azure Blob Storage API documentation <https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api>`_.
+
+Before you begin
+^^^^^^^^^^^^^^^^
+Before using Blob Storage within Airflow you need to authenticate your account with Token, Login and Password.
+Please follow Azure
+`instructions <https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal>`_
+to do it.
+
+TOKEN should be added to the Connection in Airflow in JSON format, Login and Password as plain text.
```

Comment:
Since two connections need to be established for this operator to function, it's not entirely clear to which connection `TOKEN` should be added.

Let's be more explicit that both a WASB and SFTP connection need to be created. You could also link to the existing connection docs (see [WASB](https://airflow.apache.org/docs/apache-airflow-providers-microsoft-azure/stable/connections/wasb.html) and [SFTP](https://airflow.apache.org/docs/apache-airflow-providers-sftp/stable/connections/sftp.html)) for users to reference. Perhaps callout what attributes need to be specified in the respective connections to use the operator.

Reply:
please, @josh-fell, review this change to check if it is better